Gruntfile.js

```diff
@@ -108,8 +108,12 @@ module.exports = function(grunt) {
 			],
 			js: [
 				WORKING_DIR + 'wp-admin/js/',
+				WORKING_DIR + 'wp-includes/assets/',
 				WORKING_DIR + 'wp-includes/js/'
 			],
+			assets: [
+				WORKING_DIR + 'wp-includes/js/**/*.asset.php'
```

Comment:
Based [on the core trac image](https://core.trac.wordpress.org/attachment/ticket/48154/Screen%20Shot%202019-12-04%20at%2016.10.08.png) the `php` assets are now stored in the `/src/wp-includes/assets/dist` folder

Reply:
It's more complex than that.

This is where webpack generates those PHP assets, we move them to `/src/wp-includes/assets/dist` or `/build/wp-includes/assets/dist` depending where webpack operates. It's even more hard to follow when you take into account that `npm run build:dev`  operates in `src` folder, but `npm run build` uses `build` folder ...

Reply:
This name should probably be updated to `webpack-generated-assets` :)

Reply:
@ntwb I included some tweaks to make it easier to follow. I hope it helps:

https://github.com/WordPress/wordpress-develop/pull/102/commits/ba91cc12103ffc7d0720c0c9be0aa850d47ce5cd